container_linux.go

```diff
@@ -139,10 +139,11 @@ func (c *linuxContainer) commandTemplate(p *Process, childPipe *os.File) (*exec.
 	if cmd.SysProcAttr == nil {
 		cmd.SysProcAttr = &syscall.SysProcAttr{}
 	}
-	cmd.ExtraFiles = []*os.File{childPipe}
 	// NOTE: when running a container with no PID namespace and the parent process spawning the container is
 	// PID1 the pdeathsig is being delivered to the container's init process by the kernel for some reason
 	// even with the parent still running.
+	cmd.ExtraFiles = append([]*os.File{childPipe}, p.ExtraFiles...)
+	cmd.SysProcAttr.Pdeathsig = syscall.SIGKILL
```

Comment:
I think this is a failed rebase and needs to be removed.  Also the comment above cmd.ExtraFiles needs to be on the if block for the ParentDeathSignal.